src/styles/styles.js

```diff
@@ -490,6 +552,47 @@ const styles = {
         textAlignVertical: 'center',
     },
 
+    expensiPickerContainer: {
+        borderWidth: 1,
+        borderRadius: variables.componentBorderRadiusNormal,
+        borderColor: themeColors.border,
+        justifyContent: 'center',
+        backgroundColor: themeColors.componentBG,
+    },
+    expensiPickerContainerOnFocus: {
+        borderColor: themeColors.borderFocus,
+    },
+    expensiPickerContainerDisabled: {
+        backgroundColor: colors.gray1,
+        color: colors.gray3,
+    },
+    expensiPickerLabel: {
+        fontSize: variables.fontSizeLabel,
```

Comment:
Just noticing a pattern here ... the following styles are all prettymuch the same:

- `formHint`
- `popoverMenuDescription`
- `settingsLoginName`

So it looks like we could create a reusable style like this:

```javascript
labelLabelSupporting: {
    fontFamily: fontFamily.GTA,
    fontSize: variables.fontSizeLabel,
    color: themeColors.textSupporting,
},
```

Then eliminate or greatly simplify the following styles:

- `formHint`
- `popoverMenuDescription`
- `settingsLoginName`
- `optionAlternateText`
- `expensiPickerLabel`

Reply:
Done

Reply:
I like the idea of combining styles, but why the name `labelLabelSupporting`? Does two "Label"s sounds weird or am I reading the variable name wrong?

Reply:
Hahah IDK what I was thinking there ... probably `textLabelSupporting` to mirror `textLabel`, `textMicro`, `textMicroSupporting`, etc...